docs/tutorial/hyperfoil.adoc

```diff
@@ -1,129 +1,153 @@
 = Scripting hyperfoil with quarkus getting started
 
-The `curl` command from link:quarkusgetstarted.adoc[Scripting Quarkus getting started] was a substitute for a load driver. We are going to use hyperfoil to drive load to our quarkus application and use qDup `setup-scripts` and `cleanup-scripts` phases to coordinate the test.
+The `curl` command from link:quarkusgetstarted.adoc[Scripting Quarkus getting started] was a substitute for the load generator. We are going to use link:https://hyperfoil.io[Hyperfoil] to drive the load to our quarkus application and use qDup phases to coordinate the test.
 
 == Introduction
 
-This tutorial will introduce us to:
+This tutorial will introduce:
 
-1. Commands that change the ssh shell prompt
-2. qDup `setup-scripts` and `cleanup-scripts`.
-3. qDup `roles`
+1. link:./../reference/command/addprompt.adoc[qDup add-prompt]
+2. link:./../reference/roles.adoc[qDup roles]
 
+== Prerequisites
 
-== Where we left off
+qDup connects to computers through ssh. You need to have a computer that will accept ssh connections. You can setup ssh on your own computer by following the steps in the link:./prerequisites.adoc[prerequisites]
 
-The previous tutorial created a qDup yaml that downloaded Quarkus getting started and ran a single curl against the `quarkus dev` process. We are going to start where it left off:
-
-```yaml
+== Running the experiment
+The script will clone the `quarkus-quickstarts` repository, install `quarkus-cli` if needed, and, start the application. In parallel, it will download and install link:https://hyperfoil.io[Hyperfoil]. Once the application is ready, it will start a simple benchmark. In the end, it will upload the result and stop the `quarkus-quickstarts` application.
```

Comment:
> it will upload the result 
I think you mean it will download the result to qDup

Reply:
Done